spec/authority_spec.cr

```diff
@@ -15,6 +15,9 @@ module PlaceOS::Model
       authority = Authority.new
       authority.domain = "https://localhost:8080"
       authority.domain.should eq "localhost"
+
+      authority.domain = "test.somewhere.com"
+      authority.domain.should eq "test.somewhere.com"
```

Comment:
this also works

```crystal
 URI.parse("/wrong").path # => “/wrong”
```

Reply:
yep, but that is fine, it won't break anything if someone enters that - basically the same as entering the wrong domain

Reply:
like it I enter `my.doman` instead of `my.domain` it has the same impact as entering `/obviously-wrong`

Reply:
do we want to add validation? or is it not worth it?

Reply:
mostly just not sure it's needed, 0 impact if they enter the wrong thing outside of their domain not working and they can enter the wrong thing even with extensive validation
can't see any malicious vectors as all we do is compare this with a host header